src/openrct2/interface/console.c

```diff
@@ -727,6 +727,12 @@ static sint32 cc_get(const utf8 **argv, sint32 argc)
         if (strcmp(argv[0], "park_rating") == 0) {
             console_printf("park_rating %d", gParkRating);
         }
+        else if (strcmp(argv[0], "park_value") == 0) {
+            console_printf("park_value %d", gParkValue);
```

Comment:
You'll probably want to divide both variables by 10. Otherwise, `park_value` shows as `344430` instead of `34443`, for example.

Reply:
Displaying it as a properly formatted money amount is better IMO. In that case, I want to include gCash in that as well (which currently uses a ghetto solution).

Reply:
First of all, I devided by 10.